pkg/hintrunner/zero/zerohint_ec.go

```diff
@@ -386,3 +386,49 @@ func createEcDoubleSlopeV1Hinter(resolver hintReferenceResolver) (hinter.Hinter,
 
 	return newEcDoubleSlopeV1Hint(point), nil
 }
+
+func newReduceV1Hinter(x hinter.ResOperander) hinter.Hinter{
+	return &GenericZeroHinter{
+		Name:"reduceV1",
+		Op: func(vm *VM.VirtualMachine, ctx *hinter.HintRunnerContext) error {
+			//> from starkware.cairo.common.cairo_secp.secp_utils import SECP_P, pack
+			//> x = pack(ids.x, PRIME) % SECP_P
+
+			secPBig, ok := secp_utils.GetSecPBig()
+			if !ok{
+				return fmt.Errorf("GetSecPBig failed")
+			}
+			xAddr, err := x.GetAddress(vm)
+			if err != nil {
+				return err
+			}
+
+			xMemoryValues, err := hinter.GetConsecutiveValues(vm, xAddr, int16(3))
+            if err != nil {
+                return err
+            }
+			var xValues [3]*fp.Element
+            for i := 0; i < 3; i++ {
+                xValue, err := xMemoryValues[i].FieldElement()
+                if err != nil {
+                    return err
+                }
+                xValues[i] = xValue
+            }
+			xBig, err := secp_utils.SecPPacked(xValues)
+            if err != nil {
+                return err
+            }
+			xBig.Mod(&xBig, &secPBig)
+			return ctx.ScopeManager.AssignVariable("x", xBig)
+		},
+	}
+}
+
+func createReduceV1Hinter(resolver hintReferenceResolver)(hinter.Hinter, error){
+	x, err := resolver.GetResOperander("x")
+	if err != nil {
+		return nil, err
+	}
+	return newReduceV1Hinter(x), nil
+}
```

Comment:
newline missing at the end of the file. This will probably make the linting CI check fail. Try it locally doing `make lint` so you can check if there are any problems. For issues like this one you can run `gofmt -w .` on the root of the project and the files should get formatted correctly

Reply:
For some reason, `gofmt -w` is failing for me with error `error: cannot use -w with standard input`. `make lint` pointed out the error and it worked for me by `gofmt -s -w ./pkg/hintrunner/zero/hintcode.go`

Reply:
For some reason, `gofmt -w `is failing for me with error error: cannot use -w with standard input. `make lint `pointed out the error and it worked for me by `gofmt -s -w ./pkg/hintrunner/zero/hintcode.go`